### ISSUE TITLE
Label colors and label interpolator

### ISSUE THREAD

**Issue by kephale** [kephale/cryocanvas#1]
@uermel suggested:

- change label colors so they are more distinct and better suited for cryoEM data
- use Lorenzo's label interpolator

**Comment by kephale** [comment]
Label colors were implemented in https://github.com/kephale/cryocanvas/commit/884bfe7daefc1362983eeba07f66d60b28675292

**Comment by kephale** [comment]
@uermel I'm up for using https://github.com/brisvag/napari-label-interpolator but it isn't 100% clear to me how it would be used. I can imagine using the interpolator to paint additional pixels between some existing painted regions of the same label, is that what you were thinking about? (e.g. paint the top, middle, and bottom of a ribosome, then fill in the rest) It looks like we might need to paint per-instance to get this to work as intended.

**Comment by uermel** [comment]
Yes, that's what I meant, but mainly for the membrane case. It's more tedious to trace long sections of membrane on each slice than to pick particles with the 3D sphere paint tool.

**Comment by kephale** [comment]
@uermel also wanted to change individual colors. There is a related napari zulip thread that is active right now: https://napari.zulipchat.com/#narrow/stream/212875-general/topic/Changing.20a.20single.20label.20color.20in.20a.20.60DirectLabelColormap.60
